card/db/schema.rb

```diff
@@ -66,22 +65,22 @@
   add_index "card_revisions", ["creator_id"], name: "revisions_created_by_index", using: :btree
 
   create_table "cards", force: :cascade do |t|
-    t.string   "name",                limit: 255,      null: false
-    t.string   "key",                 limit: 255,      null: false
+    t.string   "name",                limit: 255,   null: false
+    t.string   "key",                 limit: 255,   null: false
     t.string   "codename",            limit: 255
     t.integer  "left_id",             limit: 4
     t.integer  "right_id",            limit: 4
     t.integer  "current_revision_id", limit: 4
-    t.datetime "created_at",                           null: false
-    t.datetime "updated_at",                           null: false
-    t.integer  "creator_id",          limit: 4,        null: false
-    t.integer  "updater_id",          limit: 4,        null: false
+    t.datetime "created_at",                        null: false
+    t.datetime "updated_at",                        null: false
+    t.integer  "creator_id",          limit: 4,     null: false
+    t.integer  "updater_id",          limit: 4,     null: false
     t.string   "read_rule_class",     limit: 255
     t.integer  "read_rule_id",        limit: 4
     t.integer  "references_expired",  limit: 4
-    t.boolean  "trash",                                null: false
-    t.integer  "type_id",             limit: 4,        null: false
-    t.text     "db_content",          limit: 16777215
+    t.boolean  "trash",                             null: false
+    t.integer  "type_id",             limit: 4,     null: false
+    t.text     "db_content",          limit: 65535
```

Comment:
Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.